Formula/circleci.rb

```diff
@@ -1,22 +1,30 @@
 class Circleci < Formula
+  VERSION = "0.1.1430"
+  SHA     = "9788d75"
+
   desc "Enables you to reproduce the CircleCI environment locally"
   homepage "https://circleci.com/docs/2.0/local-cli/"
-  url "https://github.com/circleci/local-cli/releases/download/v0.1.0/circleci_v0.1.0.tar.gz"
-  sha256 "e6b86954c8ccb38ca16b77cfcd642a1be20c9fa471ed811c49f7105bcb96739c"
-
-  bottle :unneeded
+  url "https://github.com/CircleCI-Public/circleci-cli/archive/v#{VERSION}.tar.gz"
+  sha256 "b2dabe90c49fb6c34ee0ceb488b39487f0a1ab0ffc04fe8857ef4b0686749bf0"
 
+  depends_on "go" => :build
   depends_on "docker"
 
   def install
-    bin.install "circleci.sh" => "circleci"
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/CircleCI-Public/circleci-cli"
+    dir.install buildpath.children
+
+    ENV.append "LDFLAGS", "-s -w -X github.com/CircleCI-Public/circleci-cli/version.Version=#{VERSION} -X github.com/CircleCI-Public/circleci-cli/version.Commit=#{SHA}"
+
+    cd dir do
+      system "go", "build", "-o", bin/"circleci"
+      prefix.install_metafiles
+    end
   end
 
   test do
     # assert basic script execution
-    assert_match version.to_s, shell_output("#{bin}/circleci --version")
-    # assert script fails for missing docker (docker not on homebrew CI servers)
-    output = shell_output("#{bin}/circleci config validate 2>&1", 1)
```

Comment:
Why is this being removed?

Reply:
We changed the command, but I just added it back